### PR TITLE
replace system font stack with system-ui

### PR DESCRIPTION
**Pull request by djfarly**
see: https://caniuse.com/font-family-system-ui

### CODE REVIEW

File: sessions/react-basics/demo-inner-html-vs-render/public/styles.css [hunk 1]
```diff
@@ -6,8 +6,7 @@
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
-    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  font-family: system-ui;
```
Comment (PReichetanz) [inline sessions/react-basics/demo-inner-html-vs-render/public/styles.css:10]: Is there a reason for this empty line?
```suggestion
```

File: sessions/react-basics/demo-render/public/styles.css [hunk 2]
```diff
@@ -6,8 +6,7 @@
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
-    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  font-family: system-ui;
```
Comment (PReichetanz) [inline sessions/react-basics/demo-render/public/styles.css:10]: Is there a reason for this empty line?
```suggestion
```

File: sessions/react-state/demo-food-order-start/src/styles.css [hunk 3]
```diff
@@ -6,8 +6,7 @@
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
-    Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  font-family: system-ui;
 }
 
 #root {
```
Comment (PReichetanz) [inline sessions/react-state/demo-food-order-start/src/styles.css:12]: Is this correct?
```suggestion
:root {
```

**Review by PReichetanz — APPROVED** [review]
Checked all changed files and found some minor issues; checked three examples in CSB as well: `font-family: system-ui` is working (ckecking off the mark in the inspector changes to font-family).

If this is how I was supposed to check this PR, then LGTM! 🚀

File: sessions/react-state/demo-food-order-start/src/styles.css (hunk 3 above)
Comment (djfarly) [reply sessions/react-state/demo-food-order-start/src/styles.css:12]: yea i think this selects the `#root` element from CSB

**Pull request merged**